llvm/test/CodeGen/AArch64/ptrauth-call.ll

```diff
@@ -167,6 +167,28 @@ define i32 @test_tailcall_ib_var(ptr %arg0, ptr %arg1) #0 {
   ret i32 %tmp1
 }
 
+define void @test_tailcall_omit_mov_x16_x16(ptr %t) {
```

Comment:
[nit] Merely for uniformity:
```suggestion
define void @test_tailcall_omit_mov_x16_x16(ptr %t) #0 {
```

Reply:
Fixed, thanks, see 0cae7ea47543c6fc115ab9aa3faceb4627442832